hstream.proto

```diff
@@ -128,10 +129,16 @@ enum SpecialOffset {
   LATEST = 1;
 }
 
+message TimestampOffset {
+  int64 timestampInMs = 1;
+  bool strictAccuracy = 2;
+}
+
 message ShardOffset {
   oneof offset {
     SpecialOffset specialOffset = 1;
     RecordId recordOffset = 2;
+    TimestampOffset timestampOffset = 3;
```

Comment:
Does this have a default offset in the server?

Reply:
No, offset is specified in the request sent by the client, server just gets offset from rpc request